Makefile

```diff
@@ -113,7 +113,7 @@ build: check-git deps $(PROMU)
 
 .PHONY: crossbuild
 crossbuild: ## Builds all binaries for all platforms.
-crossbuild: deps $(PROMU)
+crossbuild: | $(PROMU)
```

Comment:
Pipe makes sure that dependency doesn't run if the file already exists.
https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html

Reply:
I think even without pipe that is happening. I guess it does not check mod file?

Reply:
I have tested locally somehow, it still tries to execute bingo install task. Just to be sure.